collection-scripts/gather

```diff
@@ -49,3 +49,12 @@ done
 
 # Gather Windows Kubernetes component logs
 /usr/bin/gather_windows_node_logs
+
+# Gather OVN traces 
+/usr/bin/gather_network_ovn_trace
+
+#Gather network logs 
+/usr/bin/gather_network_logs
+
+#Gather node core dumps 
+/usr/bin/gather_core_dumps
```

Comment:
Reading over this script (as part of the PR). This would run on all nodes, so if you had 200 nodes you would have to wait for all 200 nodes to collect this data. 

As a result I don't think we want this to run by default.

Reply:
+1 I deactivated by default